app/containers/CustomHeader/index.tsx

```diff
@@ -0,0 +1,94 @@
+import React, { ReactNode } from 'react';
+import { PixelRatio, Text, View } from 'react-native';
+import { NativeStackHeaderProps } from '@react-navigation/native-stack';
+import { HeaderBackButton } from '@react-navigation/elements';
+
+import HeaderContainer from './components/HeaderContainer';
+import { isAndroid } from '../../lib/methods/helpers';
+import { useAppSelector } from '../../lib/hooks';
+import { useTheme } from '../../theme';
+import { styles } from './styles';
+
+interface IHeaderTitle {
+	headerTitle?: string | ((props: { children: string; tintColor?: string }) => ReactNode);
+}
+
+const HeaderTitle = ({ headerTitle }: IHeaderTitle) => {
+	const { colors } = useTheme();
+	if (!headerTitle) return null;
+	if (typeof headerTitle === 'string') {
+		if (isAndroid) {
+			return (
+				<Text
+					numberOfLines={1}
+					style={{
+						...styles.title,
+						paddingVertical: 10,
```

Comment:
You should either have `./styles` or not. Having both is kinda awkward.

_I know the current state of things isn't perfect, but let's try not to create new things with past mistakes 👀_